pkg/controller/artifacts/entrypoint.go

```diff
@@ -27,25 +27,12 @@ datasources:
 - name: prometheus
   type: prometheus
   access: proxy
-  {{- if eq .DSType "common-service" }}
-  url: https://{{ .PrometheusFullName }}:{{ .PrometheusPort }}
-  {{- end }}
-  {{- if eq .DSType "openshift" }}
   url: http://127.0.0.1:9096
```

Comment:
Where does this point to?, i.e. what listens on localhost:9096?

Reply:
as per ibm-grafana-ocpthanos-proxy repo(https://github.com/IBM/ibm-grafana-ocpthanos-proxy) readme file
The address ibm-grafana-ocpthanos-proxy should listen on. Default value: 127.0.0.1:9096

This configuration will be present in datasource.yaml file inside grafana container

```
bash-4.4$ pwd
/etc/grafana/provisioning/datasources
bash-4.4$ ls -l
total 4
-rw-r--r--. 1 1000620000 1000620000 192 Apr 29 09:17 datasource.yaml
bash-4.4$ Tirumalas-MBP:IBM tirumalamannaru$
```